test/test_utils.py

```diff
@@ -1198,6 +1219,68 @@ def test_build_eager(self):
         for message, value, expected in test_data:
             self.assertEqual(expected, build_eager(value), message)
 
+    def test_compute_hash(self):
+        self.assertEqual("4de279c82056603e91aab3930a593b8b887d9e48",
+                         compute_hex_hash("https://cloudinary.com/images/old_logo.png"))
+
+        original_value = str(uuid.uuid4())
+
+        self.assertEqual(compute_hex_hash(original_value), compute_hex_hash(original_value),
```

Comment:
These tests seems a bit redundant (consistency), I would add more tests for edge cases( empty, None, int values, bytes, etc)